Open Judge System/Data/OJS.Data.Models/Contest.cs

```diff
@@ -91,6 +91,8 @@ public class Contest : DeletableEntity, IValidatableObject, IOrderable
         public string Description { get; set; }
 
         public bool UsersCantSubmitConcurrently { get; set; }
+
+        public bool EnsureValidAuthorSubmisions { get; set; }
```

Comment:
Why do we need this, don't we want all contests to be validated for author solutions?

Reply:
Extracted a property for it, to allow for granular control, on whether to run the checks or not. The reasons being that: 
- even if the requests are fast, we'll still making an extra request every time someone opens Contest -> Edit, and we might not want to make these extra queries for older contests or after a contest completes
- to allow for admins to manually disable the new feature in case there is a problem.

Reply:
Ok, but at least we should make it `true` by default, to not defeat the whole point of this task, if this is forgotten by the contest creators.
For the 2nd reason - if it's just a warning, we should not worry about it. Even more this is only visible in the administration section of the contest, not for regular users. That being said, I wonder if this is the best place for the warning. What if someone changes the tests, but does not go to the Edit page of the contest? I think this is a valid scenario...

Reply:
Good point, talked this point out with Acho as well and yes, should be turned on by default, I'll make the changes